hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreScanner.java

```diff
@@ -157,6 +158,15 @@
   private final ReentrantLock flushLock = new ReentrantLock();
   // lock for closing.
   private final ReentrantLock closeLock = new ReentrantLock();
+  // scanner reset lock - ensure scanners are reopened only on new store files(non compacted)
+  private final ReentrantLock SCANNER_RESET_LOCK = new ReentrantLock();
```

Comment:
Generally these are not static final variables. So you can follow the normal variable naming convention here. Need not be in CAPS.

Reply:
Sure, will change this